marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -440,8 +441,8 @@ const AdvancedSettingsTooltips = {
   targetFormat: 'The format of the records in the target database.',
   additionalCollections: 'The collection tags to add to the default tags assigned to the processed record.',
   targetPermissions: (
-    <span>A comma-delimited string that defines permissions required to access the processed record. The string must be in the format 
```

Comment:
@Sanjeevani19 though the slider is disabled, the tooltip number that we see while sliding over the bar is still enabled and visible. Is this expected?